### PR TITLE
Option for options... always :)

### PR DESCRIPTION
**Pull request by filiptronicek**
## Description

Context [on Slack](https://gitpod.slack.com/archives/C05K08WRCQ7/p1696572581009009)

image

## How to test

Load the updated extension and in the settings, you should see a new option for toggling this behavior.

Fixes EXP-776

### CODE REVIEW

**Comment by filiptronicek** [comment]
Maybe makes sense to remove the whole dropdown from the button in this case?

File: src/popup.tsx [hunk 1]
```diff
@@ -70,6 +71,12 @@ function IndexPopup() {
           checked={automaticallyDetect}
           onChange={setAutomaticallyDetect}
         />
+        <CheckboxInputField
+          label="Always start with options"
```
Comment (selfcontained) [inline src/popup.tsx:75]: If we decide to use this in favor of the dropdown / secondary "start with options" (I think we should), maybe rewording it for more clarity, since we have more space in this UI vs the button. Personally, I think the default behavior would be best to not autostart, and then the setting lets you opt in to auto starting. Then something like:

> Autostart workspaces

**Review by svenefftinge — APPROVED** [review]
